### PR TITLE
rgw: clean some redundant codes

### PR DESCRIPTION
**Pull request by weiqiaomiao**
Signed-off-by: weiqiaomiao wei.qiaomiao@zte.com.cn

### CODE REVIEW

File: src/rgw/rgw_rados.cc [hunk 1]
```diff
@@ -8509,7 +8509,7 @@ int RGWRados::set_attrs(void *ctx, rgw_obj& obj,
     } else {
       int ret = index_op.cancel();
       if (ret < 0) {
-        ldout(cct, 0) << "ERROR: comlete_update_index_cancel() returned r=" << r << dendl;
+        ldout(cct, 0) << "ERROR: comlete_update_index_cancel() returned ret=" << ret << dendl;
```
Comment (mattbenjamin) [inline src/rgw/rgw_rados.cc]: @weiqiaomiao s/comlete/complete/ ?

**Comment by weiqiaomiao** [comment]
@mattbenjamin  done

**Comment by cbodley** [comment]
retest this please

File: src/rgw/rgw_rest_swift.cc [hunk 2]
```diff
@@ -652,15 +652,13 @@ static int get_delete_at_param(req_state *s, real_time *delete_at)
 
   if (x_delete.empty()) {
     x_delete = s->info.env->get("HTTP_X_DELETE_AT", "");
+    return 0;
```
Comment (yehudasa) [inline src/rgw/rgw_rest_swift.cc]: @weiqiaomiao this looks wrong to me, x_delete might not be empty after this, so we miss later parsing

Comment (weiqiaomiao) [inline src/rgw/rgw_rest_swift.cc]: @yehudasa Thank for you reminding me.  I think I made a mistake, and the code had been rolled back.

File: src/rgw/rgw_rest_swift.cc [hunk 3]
```diff
@@ -661,6 +661,7 @@ static int get_delete_at_param(req_state *s, real_time *delete_at)
   if (x_delete.empty()) {
     return 0;
   }
+
```
Comment (xiexingguo) [inline src/rgw/rgw_rest_swift.cc]: I think you shall drop this line too.

Comment (weiqiaomiao) [inline src/rgw/rgw_rest_swift.cc]: @xiexingguo OK. Thanks!

**Comment by dachary** [comment]
jenkins test this please (no logs)

**Comment by theanalyst** [comment]
test this please

**Comment by oritwas** [comment]
jenkins test this please

**Comment by weiqiaomiao** [comment]
already fixed it on master branch.

**Pull request closed without merging**